azurerm/resource_arm_storage_account.go

```diff
@@ -253,6 +253,10 @@ func resourceArmStorageAccountCreate(d *schema.ResourceData, meta interface{}) e
 
 	// AccessTier is only valid for BlobStorage accounts
 	if accountKind == string(storage.BlobStorage) {
+		if string(parameters.Sku.Name) == string(storage.StandardZRS) {
```

Comment:
Would this check be needed in `Update` too as `account_replication_type` is not `ForceNew`?

Reply:
good point, most of the Azure resources are `CreateOrUpdate` - so I'd assumed this was the same - I'll update it now..